### PR TITLE
Update the README to make it more friendly

### PR DESCRIPTION
**Pull request by iankronquist**
It's basically a collection of important links to the wiki.

Features:
1. Copy stolen directly from the wiki explaining what daala is.
2. Links to various quick starts on the wiki.
3. Links to Are We Compressed Yet.
4. The rainbow barfing tiger from the IRC channel topic.
5. Links to the irc channel via webchat and the mailing list.

### CODE REVIEW

**Comment by iankronquist** [comment]
Fixes #122.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+other contributors that wish to help. The goal of the project is to provide a
+video format that's free to implement, use and distribute, and a reference
+implementation with technical performance superior to H.265. 
+For more information, see the [Daala wiki][wiki].
+
+To get a sense of how well Daala performs in comparison to other codecs, check
+out [Are We Compressed Yet?][awcy].
+
+[awcy]: https://arewecompressedyet.com/
+[wiki]: https://xiph.org/daala/
```
Comment (n3y28eaeju) [inline README.md]: This isn't a wiki redirect yet. You should just link to https://wiki.xiph.org/Daala if you want to link to the wiki.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+![Tiger barfing rainbows](https://people.xiph.org/~greg/a.png)
+
+## Hacking on Daala
```
Comment (n3y28eaeju) [inline README.md]: I'd probably put these links even before the others. If someone has the code, they want to hack.

I'd also add the other links from <https://mailarchive.ietf.org/arch/msg/hackathon/PnbblctpV-YrP5ASgXVgu-lumsk>, specifically the issue tracker, code review, and continuous integration links.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+The wiki has quickstart pages for [Linux/OS X][posix], [Windows][win], and 
+[Windows MinGW][mingw].
```
Comment (n3y28eaeju) [inline README.md]: Should probably make explicit that this is MinGW64, not just MinGW generically.

File: README.md (hunk 2 above)
Comment (iankronquist) [inline README.md]: I designed the README to be friendly to people who may not know what Daala is and why it's awesome, as well as experienced hackers. The newbies would like to see are we compressed yet, the wiki, and the barfing tiger. The old veterans are going to jump straight to the part they care about anyway.

I definitely made all the other changes you recommended though. Thanks!

File: README.md (hunk 1 above)
Comment (iankronquist) [inline README.md]: Fixed. I'm not sure why GitHub is still showing this.

**Comment by tterribe** [comment]
r=me

**Comment by iankronquist** [comment]
It looks like this has been merged. Thanks guys!

**Pull request closed without merging**